spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are represented by <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are compared using ASCII case-insensitive comparisons.
+          Time zones supported by an ECMAScript implementation may be <dfn variants="available named time zone">available named time zones</dfn>, represented by the [[Identifier]] field of the Time Zone Identifier Records returned by AvailableNamedTimeZoneIdentifiers, or <dfn variants="offset time zone">offset time zones</dfn>, represented by Strings for which IsTimeZoneOffsetString returns *true*.
+        </p>
+        <p>
+          A <dfn variants="primary time zone identifiers">primary time zone identifier</dfn> represents the preferred identifier of an available named time zone, while a <dfn variants="non-primary time zone identifiers">non-primary time zone identifier</dfn> is an alias for the same available named time zone.
+          Each available named time is represented by exactly one primary time zone identifier and zero or more non-primary time zone identifiers.
+          The set composed of all primary and non-primary time zone identifiers that are supported by an ECMAScript implementation are called <dfn variants="available named time zone identifier">available named time zone identifiers</dfn>.
```

Comment:
```suggestion
          The  <dfn variants="available named time zone identifier">available named time zone identifiers</dfn> is the set of all primary time zone identifiers and non-primary time zone identifiers that are supported by an ECMAScript implementation.
```

Reply:
@michaelficarra did you intend to strip the word "available" from these definitions? If so, why? 402 uses "available" to mean "supported by the implementation" so I was trying to follow its lead here. 

Also, it's helpful to have some differentiator between "named time zone" which could be any time zone with a name (including zones not supported by the implementation) and "available named time zone" which is a named time zone actually supported by the implementation. 

So we could change "available" to something else, but I'm not sure it's wise to remove it completely.

Reply:
> did you intend to strip the word "available" from these definitions?

No, that was just an oversight.